inspirehep/modules/records/receivers.py

```diff
@@ -183,6 +183,15 @@ def enhance_after_index(sender, json, *args, **kwargs):
     populate_inspire_document_type(sender, json, *args, **kwargs)
     populate_name_variations(sender, json, *args, **kwargs)
     populate_title_suggest(sender, json, *args, **kwargs)
+    populate_citations_count(sender, json, *args, **kwargs)
+
+
+def populate_citations_count(sender, json, *args, **kwargs):
+    """Populate citations_count in ES from"""
+    if hasattr(sender, 'get_citations_count'):
```

Comment:
this will apply to all records, is it what we want?

Reply:
I'm not sure, Its like that from Monday when I changed it to work with data also. I think that it won't hurt if all entries would have info about citations even if they won't have any.